Server/src/utils/cr_stripper.py

```diff
@@ -0,0 +1,18 @@
+
+# This utility strips carriage return characters (\r) from bytes or strings 
+class CRStripper:
```

Comment:
Can you:

- Remove the empty newline at the top
- Replace the Python comment with an actual docstring for the class
- Add type hints for `stream`, `data` and `name` parameters
  - If there are more than one type, use `|` instead of `Optional`/`Union` please e.g. `int | None` instead of `Optional[int]`